packages/flutter/test/material/theme_test.dart

```diff
@@ -298,6 +298,29 @@ void main() {
     expect(glyphText.text.style.fontSize, 20.0);
   });
 
+  testWidgets('hint color has correct default', (WidgetTester tester) async {
```

Comment:
We should be checking the a11y contrast guidelines here. Look for similar tests that use `meetsGuideline(textContrastGuideline)`